### DISCUSSION TITLE
[Feature request] Show alert when something could have been done with a shortcut

### DISCUSSION THREAD

**Discussion by DenisBessa** [Anze/KeyCluCask#42] [Ideas]
Hi! This is a great app, thanks for the amazing job!

Was wandering if KeyClu could make an option to monitor when the user do something that could be done with a shortcut "manually", and then display some alert or display.

Personally, I would like the program to have a very aggressive behavior, like freezing my screen for a second or two, to "punish" me for not knowing the correct shortcut. It would be an opportunity to force the user to learn, which is good (I, for example, was used to Windows shortcuts, and miss knowing them in MacOS).

**Comment by Anze** [comment]
Hey [DenisBessa](https://github.com/DenisBessa),
Thanks for submitting your feature request. I'm not entirely sure if it's possible at the moment, as it seems like it might be a significant addition to the project scope. If more people express interest in this feature, I would be open to exploring the possibility of implementing it.
